packages/next/server/web/spec-extension/cookies.ts

```diff
@@ -35,7 +35,7 @@ const deserializeCookie = (input: Request | Response): string[] => {
 
 const serializeCookie = (input: string[]) => input.join(', ')
 
-export class Cookies extends Map<string, string> {
+export class Cookies extends Map<string, string | object> {
```

Comment:
I do not think `Cookies` should inherit `Map`, they resemble in their form but "Cookies" are not a "Map containing either a string or an object" IMO. Both can conform to the same interface but not have any parent/child relationship between them. Maybe this should be changed before a patch/minor release is published?